acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java

```diff
@@ -246,7 +246,7 @@ public void startNode(final BesuNode node) {
     }
 
     params.add("--Xsecp256k1-native-enabled=" + node.isSecp256k1Native());
-    params.add("--Xaltbn128-native-enabled=" + node.isAltbn128Native());
+    params.add("--Xaltbn128-native-enabled=false" + node.isAltbn128Native());
```

Comment:
what are the implications of this change?

Reply:
Not sure, but that looks broken. @pinges ?

Reply:
I wonder if this is a merge fail - there was a similar change here crypto/src/main/java/org/hyperledger/besu/crypto/SECP256K1.java

Reply:
Reverted.